sigs.yaml

```diff
@@ -3583,6 +3583,43 @@ workinggroups:
     liaison:
       github: aojea
       name: Antonio Ojea
+- dir: wg-checkpoint-restore
+  name: Checkpoint Restore
+  mission_statement: >
+    This working group aims to provide a central location for the community to discuss
+    the integration of Checkpoint/Restore functionality into Kubernetes.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - API Machinery
+  - Apps
+  - Auth
+  - CLI
+  - Node
+  - Scheduling
+  label: checkpoint-restore
+  leadership:
+    chairs:
+    - github: adrianreber
+      name: Adrian Reber
+      company: Red Hat
+      email: areber@redhat.com
+    - github: haircommander
+      name: Peter Hunt
+      company: Red Hat
+      email: pehunt@redhat.com
+    - github: rst0git
+      name: Radostin Stoyanov
+      company: University of Oxford
+      email: radostin.stoyanov@eng.ox.ac.uk
+    - github: viktoriaas
+      name: Viktória Spišaková
+      company: Masaryk University
+      email: spisakova@ics.muni.cz
+  meetings: []
+  contact:
+    slack: wg-checkpoint-restore
+    mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-checkpoint-restore
```

Comment:
This is missing the steering liason field, we discussed in the steering meeting today, you can add me..

Looking forward to being more directly helpful with @rst0git @viktoriaas and crew :-)

Reply:
Added, thanks

```suggestion
    mailing_list: https://groups.google.com/forum/#!forum/kubernetes-wg-checkpoint-restore
    liaison:
      github: BenTheElder
      name: Benjamin Elder